### PR TITLE
Hasura v2: user limits and camelcase

### PR DESCRIPTION
**Pull request by droserasprout**

### CODE REVIEW

**Comment by droserasprout** [comment]
[Optionally disable aggregation queries](https://trello.com/c/0vD2OSKD/140-optionally-disable-aggregation-queries)

**Comment by droserasprout** [comment]
[Global row fetch limit for all tables](https://trello.com/c/f9b7hUFq/139-global-row-fetch-limit-for-all-tables)

**Comment by droserasprout** [comment]
[Hasura metadata: camel case](https://trello.com/c/oFt6QCIR/136-hasura-metadata-camel-case)

File: src/dipdup/hasura.py [hunk 1]
```diff
@@ -94,137 +46,424 @@ def _iter_models(*modules) -> Iterator[Tuple[str, Type[Model]]]:
...
+            for field_name, field in model._meta.fields_map.items():
+                if field.pk:
+                    filter = field_name
+                    break
+            else:
+                raise RuntimeError(f'Table `{table_name}` has no primary key. How is that possible?')
+
+            table_name = f'{self._database_config.schema_name}_{table_name}'
+            fields = await self._get_fields(table_name)
+            queries.append(self._format_rest_query(table_name, table_name, filter, fields))
```
Comment (m-kus) [inline src/dipdup/hasura.py:260]: `{table_name}_by_pk`

**Review by m-kus — APPROVED** [review]

**Pull request merged**